code/_onclick/hud/action_button.dm

```diff
@@ -179,21 +179,21 @@
 		return
 
 	var/button_number = 0
-
 	if(hud_used.action_buttons_hidden)
 		for(var/datum/action/A in actions)
 			A.button.screen_loc = null
 			if(reload_screen)
 				client.screen += A.button
 	else
 		for(var/datum/action/A in actions)
+			var/isowner = (A.owner == src)
```

Comment:
```suggestion
			var/is_owner = (A.owner == src)
```

Reply:
you could rename some vars like A and B to make stuff better to read

Reply:
yeah this is just like 2 hours of work, testing, and debugging will implement in next commit